src/ParticleEmitter_TEST.cc

```diff
@@ -91,6 +91,11 @@ void ParticleEmitterTest::CheckBasicAPI()
   math::Color    expectedColorEnd        = ignition::math::Color::White;
   double         expectedScaleRate       = 1;
   std::string    expectedColorRangeImage = "";
+  // Particle scatter ratio is stored in user data
+  // TODO(anyone) Add API to set scatter ratio
+  // (this requires adding a virtual function in the base class,
+  // which breaks ABI, so this should be done in an unreleased version)
+  Variant        emptyVariant;
```

Comment:
Would it be too late to make this update in Edifice?

Reply:
This is in the line between bug fix and new feature. So I think it's ok to address it, but let's open a PR asap, like today!

Reply:
Working on it now, I'll post a follow-up here when the PR is open

Reply:
Edit: I just realized that I can't make the PR for Edifice until we merge this :upside_down_face: so, once we merge this, I will make a PR!